.rubocop.yml

```diff
@@ -61,6 +61,11 @@ Style/CollectionMethods:
 Style/WordArray:
   Enabled: false
 
+# in rspec, expect{...} is conventional
```

Comment:
Yeah? I prefer the latter example to the former, which helps recognize the distinction between parentheses and braces, and that is our current practice in Sufia (and CurationConcerns and ActiveFedora, too, for instance). There are 0 uses of `expect{` in these codebases.